asreviewcontrib/semantic_clustering/semantic_clustering.py

```diff
@@ -43,13 +38,18 @@
 
 def SemanticClustering(asreview_data_object):
 
+    # if data folder exists, delete it
+    if os.path.exists("data"):
+        print("data folder exists, deleting...")
+        os.system("del /F /Q data")
```

Comment:
Is this the best way to do this? Looks tricky and platform-dependent.

Reply:
It is :). This type of storing files to a data folder will be removed in a later PR. I want the user to select a storage location.